dephell/commands/autocomplete.py

```diff
@@ -49,23 +50,16 @@ def __call__(self):
     def _bash(self):
         script = make_bash_autocomplete()
 
-        # https://github.com/dephell/dephell/pull/62
-        path = Path.home() / '.local' / 'etc' / 'bash_completion.d' / 'dephell.bash-completion'
-        if platform().lower() == 'darwin':
-            # ref. https://itnext.io/programmable-completion-for-bash-on-macos-f81a0103080b
-            path = Path('/') / 'usr' / 'local' / 'etc' / 'bash_completion.d' / 'dephell.bash-completion'
+        # Install completions to the correct location for modern bash-completion.
+        # This will be sourced on-demand by bash-completion as soon as dephell is
+        # completed for the first time.
+        xdg_data_home = Path(os.getenv('XDG_DATA_HOME', '~/.local/share')).expanduser()
+        bashcomp_user_dir = Path(os.getenv('BASH_COMPLETION_USER_DIR', xdg_data_home / 'bash-completion'))
```

Comment:
There are paths have reverse order of priority that quite difficult to read. I think, this will be more readable:

```python
if 'BASH_COMPLETION_USER_DIR' in os.environ:
  path = Path(os.environ['BASH_COMPLETION_USER_DIR'])
else:
  path = os.getenv('XDG_DATA_HOME')
  if not path:
    path = '~/.local/share'
  path = Path(path).expanduser() / 'bash-completion'
path = path / 'completions'
```

What do you think?

Reply:
Fixed.